Sources/web3swift/Utils/ENS/ENS.swift

```diff
@@ -107,7 +106,7 @@ public class ENS {
         guard isAddrSupports else {
             throw Web3Error.processingError(desc: "Address isn't supported")
         }
-        var options = options ?? defaultOptions
+        var options = options ?? defaultTransaction
```

Comment:
It's better now than it was before that's for sure but the API of `class ENS` that has functions which accept transactions by the name `options` is just making things look weird. 
I'm accepting this PR and will create a new issue that must be handled in v4.0 to alter that API and rename all `options` to `transaction` **at least**.

Reply:
https://github.com/web3swift-team/web3swift/issues/731

Reply:
Was thinking the same when I saw it and thought, unfortunately, I can't change it without breaking the API. 😅